pom.xml

```diff
@@ -313,6 +313,24 @@
                 </plugins>
             </build>
         </profile>
+        <!-- add JAXB to module path in Java 9 -->
+        <profile>
+            <id>java9</id>
+            <activation>
+                <jdk>9</jdk>
```

Comment:
`<jdk>[9,)</jdk>` should also make this working for JDK 10 and later. I suggest to adapt the naming of the profile ID and the comment above to say something like "Java 9 and later".